components/SpeakerCarousel.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react'
 import ReactNative, { TouchableOpacity, Text, View, Image, Dimensions, Linking } from 'react-native'
-import client, {Color} from '@doubledutch/rn-client'
+import client, {Color, Avatar} from '@doubledutch/rn-client'
```

Comment:
Avatar, used below, should set client to `client`, not `item`.

Reply:
Do you mean to drop using Avatar with a 3rd party image?